### ISSUE TITLE
Find out if MDI synoptic maps exist

### ISSUE THREAD

**Issue by dstansby** [dstansby/pfsspy#164]
In theory MDI synoptic magnetograms might exist somewhere, but they don't seem to be on JSOC: http://jsoc.stanford.edu/MDI/MDI_Magnetograms.html. If anyone knows where to get them, please post below!

**Comment by STBadman** [comment]
There are synoptic map fits files indexed by carrington rotation with filenames :

`mdi.synoptic_mr_small_96m.1914` where the last 4 numbers are the carrington rotation in question

The sunpy jsoc client can find them, and they're already accessible via `sunpy.net.Fido`!

The only hiccup is that they don't appear to have valid start or endtimes (given they are referenced only by carrington rotation number) so the `Fido.search` command returns all the maps regardless of the `a.Time` attribute, so the `Fido.fetch` call should specify this by indexing the `parfive.Results` instance.

Here is a snippet which downloads all the mdi synoptic maps to `~/sunpy/data/  

```python
from sunpy.net import Fido,attrs as a
res = Fido.search(a.Time('2002-11-06T00:00:00', '2002-11-07T00:00:00'),  
                  a.jsoc.Series('mdi.synoptic_Mr_small_96m'),  
                  a.jsoc.Notify('samuel_badman@berkeley.edu')
                 )  
downloaded_files = Fido.fetch(res)
```

**Comment by dstansby** [comment]
Yeah, the JSOC bug is fixed in SunPy `2.0` (released soon!). As with HMI (see https://github.com/dstansby/pfsspy/pull/161/files) maybe it's possible to search by Carrington rotation too.
